stellargraph/__init__.py

```diff
@@ -69,3 +71,41 @@
     "PPNPPropagationLayer": layer.ppnp.PPNPPropagationLayer,
     "APPNPPropagationLayer": layer.appnp.APPNPPropagationLayer,
 }
+
+
+def _top_level_deprecation_warning(name, path):
+    msg = (
```

Comment:
FWIW, from my perspective, I think it'd be fine to not bother wrapping the string, and just write:

```python
warnings.warn(
    f"'{name}' is no longer available at the top-level. Please use 'stellargraph.{path}.{name}' instead.",
    DeprecationWarning,
)
```

This seems slightly easier to understand/work with, but maybe PyCharm or whatever interfere?